pkg/noderesourcetopology/score.go

```diff
@@ -61,6 +62,11 @@ func (tm *TopologyMatch) Score(ctx context.Context, state *framework.CycleState,
 		return 0, nil
 	}
 
+	// if it's a non-guaranteed pod, every node is account to be a good fit
+	if v1qos.GetPodQOS(pod) != v1.PodQOSGuaranteed {
+		return 100, nil
```

Comment:
the intent of the code is well explained and looks fine to me, I just wonder if the right value to use here (100) is documented somewhere and if we should use a named constant for it.